lib/oxsdatatypes/src/date_time.rs

```diff
@@ -2039,42 +2038,24 @@ fn time_on_timeline(props: &DateTimeSevenPropertyModel) -> Option<Decimal> {
 }
 
 /// A parsing error
-#[derive(Debug, Clone)]
-pub struct ParseDateTimeError {
-    kind: ParseDateTimeErrorKind,
-}
-
-#[derive(Debug, Clone)]
-enum ParseDateTimeErrorKind {
+#[derive(Debug, Clone, thiserror::Error)]
+pub enum ParseDateTimeError {
```

Comment:
I would think that
- either we make the variant private
either we build more meaningful variants (the `Message` catch-all is quite hugly)

WDYT?